examples/etcd/etcd-in-cluster/k8s/README.md

```diff
@@ -48,9 +48,17 @@ secret is going to have the name of the format `etcd-<etcd-pod-namespace>` with
 
 - **key** : TLS key file, would be `/etc/kubernetes/pki/etcd/server.key` in case of Kubeadm cluster
 
+- **labels** : Labels using which kaniter will identify one etcd member in case of multi member etcd cluster
```

Comment:
```suggestion
- **labels** : Labels using which Kanister will identify one etcd member in case of multi member etcd cluster
```

Reply:
> Labels using which Kanister will identify one etcd member in case of multi member etcd cluster
Does this mean each pod will have a different label?